src/marked.js

```diff
@@ -127,6 +127,43 @@ marked.getDefaults = getDefaults;
 
 marked.defaults = defaults;
 
+/**
+ * Use Extension
+ */
+
+marked.use = function(extension) {
+  const opts = merge({}, extension);
+  if (extension.renderer) {
+    const renderer = marked.defaults.renderer || new Renderer();
+    for (const prop in extension.renderer) {
+      const prevRenderer = renderer[prop];
+      renderer[prop] = (...args) => {
```

Comment:
I can't remember if we we supporting IE 11 anymore or which version of Node, but this syntax might be a breaking change.

Reply:
this source will be compiled by babel to es5 in `lib/marked.js` which is currently used to support ie11.

I think we are just supporting node `lts` and `latest` which should support this syntax.